inhtasks/inhtask4.py

```diff
@@ -0,0 +1,14 @@
+class CustomException(Exception):
+    def __init__(self, msg):
```

Comment:
Msg is optional in Exception's __init__, you should use *args to pack and unpack it